R/use_r_workflows.R

```diff
@@ -29,6 +29,9 @@ use_calc_coverage <- function(workflow_name = "call-calc-coverage.yml") {
 }
 
 #' Use workflow in current pkg  to automate documenting and styling
+#'
+#' Style your R package components automatically by running devtools::document(),
+#' styler::style_pkg() and usethis::use_tidy_description().
```

Comment:
quibble: I prefer the use of the oxford comma, which is missing here

Reply:
I do, too! :)